__test__/api/comments/createSingleComment.test.js

```diff
@@ -0,0 +1,75 @@
+const app = require("../../../server");
+const supertest = require("supertest");
+const CommentModel = require("../../../models/comments");
+const request = supertest(app);
+
+describe("POST /comments", () => {
+  let comment;
+
+  afterEach(async () => {
+    await CommentModel.findByIdAndDelete(comment.commentId);
+    comment = null;
+  });
+
+  test("Should create comment", async () => {
```

Comment:
I'd also suggest you use the toEqual() matcher the way I used it in the getAllReplies test. Check it out and tell me what you think. It's just a suggestion though. This works fine too. It's just verbose.